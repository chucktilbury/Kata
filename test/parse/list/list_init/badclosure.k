;-------------------------------------------------------------------------------
; verify that bad closure causes error
;-------------------------------------------------------------------------------

var list name = [123, 123, 123
var list name ; just so it's not end of file
