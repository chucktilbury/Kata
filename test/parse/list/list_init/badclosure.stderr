Syntax: /home/chuck/Src/Kata/test/parse/list/list_init/badclosure.k: 6: 4: expected a ',' or a ']' but got var
